app/resources/applications/Applications.jl

```diff
@@ -13,7 +13,7 @@ export Application
 @kwdef mutable struct Application <: AbstractModel
   id::DbId        = DbId()
   name::String    = ""
-  port::Int       = rand(10_000:65_535)
+  port::Int       = 0
```

Comment:
This should be `Union{Int,Nothing}` or at least define an `const UNASSIGNED_PORT = 0` (but really, we should use `Nothing` to express unassigned values).